rock/python.rb

```diff
@@ -200,6 +206,12 @@ def self.activate_python(ws: Autoproj.workspace,
         [bin, version]
     end
 
+    def self.deactivate_python(ws: Autoproj.workspace)
+        remove_python_shims(ws.root_dir)
+        ws.config.reset('python_executable')
+        ws.config.reset('python_version')
```

Comment:
Should maybe save after removing the options.

Reply:
Sorry, I do not understand what you are trying to say here,

Reply:
Sorry, I thought I replied.

I am wondering if it wouldn't make sense to save the configuration file right away (`ws.config.save`). Just a wonder, really.

Reply:
That would make an exemption from the rule - in all other cases config.save is triggered after all packages sets have been loaded. So I would not do it.